### PR TITLE
Update app's level

### PR DESCRIPTION
**Pull request by kay0u**
I'm sorry, I screwed up (Typhoon Hagibis has bad effect on me).

To be confirmed:

- [x] [osada](https://ci-apps.yunohost.org/ci/job/4986)
- [x] [kresus](https://ci-apps.yunohost.org/ci/job/4813)
- [x] [framaforms](https://ci-apps.yunohost.org/ci/job/4817)
- [x] [wordpress](https://ci-apps.yunohost.org/ci/job/5005)
- [x] [wekan](https://ci-apps.yunohost.org/ci/job/5056)
- [x] [bitwarden](https://ci-apps.yunohost.org/ci/job/5068)

### CODE REVIEW

File: apps.json [hunk 1]
```diff
@@ -146,7 +146,7 @@
     },
     "bitwarden": {
         "branch": "master",
-        "level": 7,
+        "level": 0,
```
Comment (yalh76) [inline apps.json]: For bitwarden, I have an ongoing [PR](https://github.com/YunoHost-Apps/bitwarden_ynh/pull/14), if we can wait a bit. 
 :)

Comment (kay0u) [reply apps.json]: Meh, failed again

Comment (yalh76) [reply apps.json]: Let's merge, I will work on it.

**Comment by kay0u** [comment]
Bump

**Pull request merged**